### DISCUSSION TITLE
Project Category Label

### DISCUSSION THREAD

**Discussion by 0x4007** [devpool-directory/devpool-directory#476] [Ideas]
We started experimenting with using the DevPool system for roles beyond software development. 

- https://github.com/ubiquity/grants is a good example of our initiative for grant writing. 

- We also have a miscellaneous tasks repo at https://github.com/ubiquity/.github 

Perhaps in the pull request to monitor the repository, we should add a property, e.g. "category" that will automatically tag those issues with a certain customizable label. 

For every repository added so far, the category can be "Development". Perhaps we could also populate all of the extra labels for extra metadata as we sometimes label bounties as "CI/CD" or "Solidity" or "UI/UX"

---

Ultimately we want to allow non developers to contribute to DAOs and I think we might be able to get it to work with this current infrastructure.

**Comment by rndquu** [comment]
This "category" field should be optional because if we implement automatic addition of partner project URLs via bot's PR then it is tricky to get project category from the code. I mean only a partner knows the exact project category so he would need to manually create a PR which adds the "category" field [here](https://github.com/ubiquity/devpool/blob/development/projects.json).

**Reply by 0x4007** [reply]
Optional is fine!

**Reply by rndquu** [reply]
Check this issue: https://github.com/ubiquity/devpool-directory-bounties/issues/2
